assets/_scss/elements/_list.scss

```diff
@@ -21,7 +21,7 @@ article {
     }
```

Comment:
I think if you want to have unstyled lists as the default you should move these rules under a "styled" list class. I think my issue here is that it doesn't necessarily make sense that the styled list would be under an article tag.